src/ui/events.js

```diff
@@ -217,15 +223,15 @@ export class MapTouchEvent extends Event {
      *
      * @example
      * // logging features for a specific layer (with `e.features`)
-     * map.on('touchstart', 'myLayerId', function(e) {
-     *     console.log('There are', e.features.length, 'features at point', e.point');
+     * map.on('touchstart', 'myLayerId', (e) => {
+     *     console.log('There are', e.features.length, 'features at point', e.point);
      * });
      *
      * @example
      * // logging all features for all layers (without `e.features`)
-     * map.on('touchstart', function(e) {
-     *     var features = map.queryRenderedFeatures(e.point);
-     *     console.log('There are', features.length, 'features at point', e.point');
+     * map.on('touchstart', (e) => {
+     *     const features = map.queryRenderedFeatures(e.point);
+     *     console.log('There are', features.length, 'features at point', e.point);
```

Comment:
The pattern here is inconsistent. In other places, we use tagged strings. Maybe we want to standardize on that here and line 227?

Reply:
fixed in 6e35b27